src/libs/Navigation/linkingConfig.js

```diff
@@ -82,11 +82,13 @@ export default {
             IOU_Request: {
                 screens: {
                     IOU_Request_Root: ROUTES.IOU_REQUEST,
+                    IOU_Request_Currency_Root: ROUTES.IOU_REQUEST_CURRENCY,
```

Comment:
`IOU_Request_Currency_Root` > `IOU_Request_Currency`. Root is only necessary for the root screen for that navigator.